spec/models/hair_trigger_spec.rb

```diff
@@ -5,6 +5,12 @@
 RSpec.describe HairTrigger, type: :model do
   describe ".migrations_current?" do
     it "is always true" do
+      # work-around empty AR::Base descendants array caused by with_model cleanup
```

Comment:
I wonder if this is something one should fix in `with_model` directly. Maybe something I can look into during the next cooldown.